### ISSUE TITLE
[ONNX Conversion] Test Qwen3 conversion

### ISSUE THREAD

**Issue by EunjuYang** [nnstreamer/nntrainer#3404]
Assignee: @sumon-98  
cc: @baek2sm 
 
- Please test the ONNX conversion for the Qwen3 model.
- Please list any issues or WTD (Work to Do) encountered during the conversion process.
- Let’s begin with the list.

**Comment by sachin-nntrainer** [comment]
Hi @EunjuYang @baek2sm 

We are done with Qwen3 1.7B modelling and have exported a ONNX version. The logits match across the official, custom pytorch models and onnx model. 

However, there is a repeat_kv module in Qwen3 which adds a dimension to 4D tensor making it 5D. Expand operation is performed on the result then. We have fixed the 4D to 5D tensor issue using reshape. However we have to implement expand op in nntrainer. We are working on that. 

Shall i raise a draft PR for modelling and onnx converter ?
